Readme.md

```diff
@@ -249,11 +250,15 @@ import { AdSettings } from 'react-native-fbads';
 
 AdSettings contains global settings for all ad controls.
 
+#### deviceHashedId
```

Comment:
@jakubsta they are certainly not consistent because they also call it `deviceIdHash` and `testDeviceHash`. `deviceHashedId` looks like something where I'd easily make typos.